src/viewer/index.ts

```diff
@@ -45,6 +45,8 @@ class Viewer {
   targetZoom = 5;
   minZoomLevel = 1;
   maxZoomLevel = 10;
+  frameCount = 0;
+  raycastFrameInterval = 16;
```

Comment:
I would make both these private and make raycastFrameInterval readonly.

Reply:
We should probably be thinking more what should be private. I'll admit I still have holes in my Typescript knowledge, partly because for a while I have been the only senior on the teams I've worked on.